aries_cloudagent/protocols/didexchange/v1_0/manager.py

```diff
@@ -620,7 +620,7 @@ async def create_response(
 
         if ConnRecord.State.get(conn_rec.state) is not ConnRecord.State.REQUEST:
             raise DIDXManagerError(
-                f"Connection not in state {ConnRecord.State.REQUEST.rfc23}"
+                f"Connection not in state {ConnRecord.State.REQUEST.rfc160}"
```

Comment:
This usage is a bit weird but I think I'd probably leave it as is instead of changing to rfc160